wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,75 @@
+# How to install drivers and fix them 
+Hello, in this post you will know how to download or update your drivers for your computer.
+It's hightly recommended to don't use third-party software to update drivers.
+## Before you get into the process, you need to know your computer.
+The first step in installing drivers is to correctly find the hardware you need drivers for. This information can be found in your computer's manual or by using the Device Manager (Windows) (right click into windows icon and press “Device Manager”). In the Device Manager, look for the device you need drivers for and expand its entry to see its model number. In System Information, navigate to the “Hardware” section and find the device you need drivers for.
+
+## Method 1: Check Windows Update
+Most of the time, you will find updates for your driver.
+
+ 1. Open the configuration for Windows via windows menu or the searcher.
+ 2. Go to Updates and security.
+ 3. Click to “Check updates”.
+ 4. Windows Update might say that you are up-to-date even if optional updates are available. In the case you see optional updates, and you click into it, you will see some updates (normally drivers), you select to all, and you need to click “Download and install” button.
+
+## Method 2: Download via brand device page.
+It's highly recommended to downloading the drivers or update them via brand page or app by the fact that the company verifies that the driver works well with the computer (graphics in most cases is on the brand GPU page)
```

Comment:
Perhaps change it to this?

It's highly recommended to download or update the drivers, through the manufacturer's website, as it's guaranteed that they will work correctly. This is especially a good idea for the GPU (Graphics) drivers.